EIPS/eip-3525.md

```diff
@@ -1,54 +1,37 @@
 
 Tokenization is one of the most important trends by which to use and control digital assets in crypto. Traditionally, there are two approaches to do so: fungible and non-fungible tokens. Fungible tokens are generally powered by ERC-20 standard, where every unit of an asset is identical to each other. ERC-20 is a flexible and efficient way to manipulate fungible tokens. Non-fungible tokens are predominantly ERC-721 tokens, powered by a standard capable of distinguishing digital assets from one another based on identity. 
 
 However, both have significant drawbacks. For example, ERC-20 requires that users create a separate ERC-20 contract for each individual data structure or combination of customizable properties, resulting in an unrealistic amount of ERC-20 contracts that need to be created in practice. On the other hand, ERC-721 tokens provide no quantitative feature, hence significantly undercutting their computability, liquidity, and manageability. For example, if we are to create financial instruments such as bonds, insurance policy, vesting plans using ERC-721, no standard interfaces are available for us to control the value in them, making it impossible, for example, to transfer a portion of the equity in the contract represented by the token. 
 
-A more intuitive and straightforward way to solve the problem is to create a semi-fungible token that has the quantitative features of ERC-20 and qualitative attributes of ERC-721. The backwards-compatibility of such semi-fungible tokens would help utilize existing infrastructures already in use and lead to faster adoption.
-
-For further design motivations, please view papers and documents below:
-
-**Articles & Discussions**
-
-- [How ERC-3525 can improve position management in uniswap-v3](https://solvprotocol.medium.com/how-vnft-can-improve-position-management-in-uniswap-v3-221ab49a8cb2)
-- [What are digital assets](https://unizon.pro/en/7632.html)
-- [ERC-3525 tokens vs.erc-20 vs. erc-721](https://medium.com/solv-blog/vnft-tokens-vs-erc-20-vs-erc-721-e75843053786)
+A more intuitive and straightforward way to solve the problem is to create a semi-fungible token that has the quantitative features of ERC-20 and qualitative attributes of ERC-721. The backwards-compatibility with ERC-721 of such semi-fungible tokens would help utilize existing infrastructures already in use and lead to faster adoption.
 
 ## Specification
 
 The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
 
-**Related Standards**
-
-- [ERC-721 Non-Fungible Token Standard](./eip-721.md)
-- [ERC-165 Standard Interface Detection](./eip-165.md)
-- [JSON Schema](https://json-schema.org/)
-- [RFC 2119 Key words for use in RFCs to Indicate Requirement Levels](https://www.ietf.org/rfc/rfc2119.txt)
-
 **Every ERC-3525 compliant contract must implement the ERC3525, ERC721 and ERC165 interfaces**
```

Comment:
```suggestion
**Every ERC-3525 compliant contract must implement the ERC-3525, ERC-721 and ERC-165 interfaces**
```

Reply:
Although seems a little redundant to also declare it implements its own interface.

Reply:
Not sure whether the interface name should contain '-' or not, I remember that we had removed the '-' sometime according to EIP standard.

Reply:
If you're referring to the EIP, you should have the `-`